hub/server_test.go

```diff
@@ -390,11 +349,11 @@ func TestMetricsCollect(t *testing.T) {
 
 func TestMetricsVersionIsAccessible(t *testing.T) {
 	v := viper.New()
-	v.Set("metrics", true)
+	v.Set("metrics_enabled", true)
 	server := newTestServer(t, v)
 	defer server.shutdown()
 
-	resp, err := server.client.Get("http://" + testAddr + "/metrics") //nolint:noctx
+	resp, err := server.client.Get("http://" + testMetricsAddr + "/metrics") //nolint:noctx
```

Comment:
You could also check if `/healthz` can be accessed here.

Reply:
Done